test/oauth.spec.ts

```diff
@@ -0,0 +1,64 @@
+import "mocha";
+import {ApiCall} from "../src/routes/oath";
+//import {refresh_token} from "../src/routes/oath";
+const chai = require( 'chai')
+var assert = chai.assert;
+const axios = require("axios");
+
+const HAPIKEY = process.env.HAPIKEY;
+
+const correctCall = async () => {
+    try {
+      //  const allContacts = 'https://api.hubapi.com/contacts/v1/lists/all/contacts/all?hapikey=' + HAPIKEY;
+        const response = await axios.get('https://dog.ceo/api/breeds/list/all')
```

Comment:
I was getting errors with the all contacts call both in-app and using other programs like curl so I used a dummy API call which still demonstrates the expected behaviour.